sqlmesh/core/snapshot/evaluator.py

```diff
@@ -490,7 +484,7 @@ def _audit(
         if audit.skip:
             return AuditResult(
                 audit=audit,
-                model=snapshot.model,
+                model=snapshot.model if snapshot.is_model else None,
```

Comment:
This seems like a fairly common pattern. Can this be a `snapshot.model_or_none` instead?

Reply:
I like that!